erts/emulator/test/trace_call_time_SUITE.erl

```diff
@@ -65,6 +65,7 @@
 -export([all/0, suite/0,
 	 init_per_testcase/2, end_per_testcase/2, not_run/1]).
 -export([basic/1, on_and_off/1, info/1,
+         apply_bif_bug/1, abb_worker/1,
```

Comment:
I think `apply_bif_bug` should be added to `all/0` too.

Reply:
Yes! Thank you.